catalyst_voices/packages/internal/catalyst_voices_models/lib/src/user/account.dart

```diff
@@ -40,6 +40,14 @@ final class Account extends Equatable {
   /// transaction was not yet read.
   final bool isProvisional;
 
+  // TODO(damian-molinski): Not integrated. Backend should return it.
+  String get stakingAddress {
```

Comment:
Usually it's  called `stakeAddress`

```suggestion
  String get stakeAddress {
```